data-table/table.js

```diff
@@ -540,13 +538,58 @@ function createRowInputAndBtn(config) {
     addRowBtn.onclick = function () {
         let value = input.value.trim();
         if (value !== '') {
+            // add to dropdown
+            let opt = document.createElement('option');
+            opt.value = value;
+            opt.innerHTML = value;
+            select = document.getElementById(config.entryIds.nameInputId);
+            select.appendChild(opt);
+
             addSingleRow(config, config.numRows, value);
         } else {
             addSingleRow(config, config.numRows);
         }
         input.value = '';
     }
     entryBoxDiv.appendChild(addRowBtn);
+
+    let inputName = document.createElement("SELECT");
```

Comment:
We've been talking about this as a dropdown but that's not quite right.

Potentially just <input> elements stacked on top of each other and linked to their row so any update to their value updates the row.